assets/js/phoenix_live_view.js

```diff
@@ -1069,7 +1069,7 @@ export class View {
     this.viewHooks = {}
     this.channel = this.liveSocket.channel(`lv:${this.id}`, () => {
       return {
-        url: this.href || this.liveSocket.main.href,
+        url: this.href || this.liveSocket.href,
```

Comment:
Hrm, just to be clear, could it be simlpy `url: this.href`? Or does it break other things?

Also, just a curiosity, what does `this.liveSocket.href` refer to? Is the root or the main one?

@joenoon, can you please confirm this fix still works for you on master?

Reply:
@josevalim i've tried this change on top of master 0f1b070f337c592944efa9ae4d619fe26ef1c70c and nothing broke for me.  However, I'm no longer using nested live views - I've gone full-in on one Main live view and everything else is now components.  So I'm not sure if this would affect someone in the nested-liveview scenario.

Reply:
@josevalim sorry, just `this.href` is enough. I got the wrong change when I committed.

As far as I understood, both `this.liveSocket.href` and `this.liveSocket.root.href` looks returns the same URL.